### PR TITLE
[Loc] MWPW-161619 - minor html and css fixes

### PR DESCRIPTION
**Pull request by saurabhsircar11**
* Replaced a tag with button
* Center aligned include fragment switch and label

Resolves: https://jira.corp.adobe.com/browse/MWPW-161619

**Test URLs:**
- Before: https://stage--milo--adobecom.hlx.page/drafts/sircar/locui-create?martech=off
- After: https://mwpw-161619-minor-fixes--milo--saurabhsircar11.hlx.page/drafts/sircar/locui-create?martech=off

### CODE REVIEW

File: libs/blocks/locui-create/fragments/view.js [hunk 1]
```diff
@@ -56,7 +56,9 @@ export default function FragmentsSection({
    </ul>`
 }
    <div>
-    ${errorFragments && errorFragments.length > 0 && html`<div class='form-field-error'>Invalid fragments <a href="#" onClick=${() => showFragments(errorFragments)}>view details</a></div>`}
+    ${errorFragments && errorFragments.length > 0 && html`<div class='form-field-error'>Invalid fragments <button class='locui-invalid-fragments-btn' onClick=${() => {
+    showFragments(errorFragments);
+  }}>view details</button></div>`}
```
Comment (robert-bogos) [inline libs/blocks/locui-create/fragments/view.js:61]: Nit: You can shorten this with optional chaining.
```suggestion
    ${errorFragments?.length > 0 && html`<div class='form-field-error'>Invalid fragments <button class='locui-invalid-fragments-btn' onClick=${() => {
    showFragments(errorFragments);
  }}>view details</button></div>`}
```

**Review by robert-bogos — APPROVED** [review]
👍

**Review by robert-bogos — COMMENTED** [review]
Are the failing unit tests related to your changes?

**Comment by saurabhsircar11** [comment]
> Are the failing unit tests related to your changes?
Hi @robert-bogos ,
I looked into failing test cases, I think those aren't related to my changes, but it won't cause an issue as this branch is getting merged to milostudio-stage, it has loc-related changes.

**Review by maagrawal16 — APPROVED** [review]

**Review by hnvadobe — APPROVED** [review]

File: libs/blocks/locui-create/fragments/view.js (hunk 1 above)
Comment (saurabhsircar11) [reply libs/blocks/locui-create/fragments/view.js]: Hi @robert-bogos ,
Thanks , fixed the review comment.

**Review by nkthakur48 — APPROVED** [review]

**Pull request merged**